src/sentry.ts

```diff
@@ -208,6 +220,36 @@ export default class SentryService {
     }
   }
 
+  flattenNestedJson(dictionary: Record<string, unknown>, init_vector?: string): { [key: string]: number | string } {
```

Comment:
Remove this method since it would set more tags than needed

Reply:
Removed `flattenNestedJson ` function